projects/documentation/content/support-and-compatibility.md

```diff
@@ -13,6 +13,20 @@ This page provides comprehensive information on versioning, public APIs, browser
 
 Starting from version 1.0.0, Spectrum Web Components follows semantic versioning ([semver](https://semver.org/)). We regularly release patch versions, which do not contain breaking changes. When a breaking change occurs, it will be done in a major version release to avoid breaking existing applications depending on the old version. Major version releases will be communicated in advance, and migration guides will be provided.
 
+### Beta versions
+
+To provide early access to upcoming releases, we maintain a beta tag that points to the next minor version. The beta tag will always correspond to the next minor version incremented from the latest stable release. For example, if the latest tagged release is `1.2.1`, the beta tag will be `1.3.0-beta.0`.
```

Comment:
Is there a way we can spin up a parallel beta site with the existing netlify account?

Reply:
I added the functionality to trigger a beta website deployment on `main`.

Here is the link: https://beta--spectrum-web-components.netlify.app/